### ISSUE TITLE
[15.0] web_advanced_search: values in code editor not propagated

### ISSUE THREAD

**Issue by coffeinisfun** [OCA/web#2624]
## Module
web_advanced_search

## Describe the bug
When using the advanced search feature in Debug Mode changes in the Code Editor are not propagated to the upper section with clickable expression lists
e.g. in the screenshot the value '7' should be propagated into the upper section and replace '1'

![web_advanced_search_bug](https://github.com/OCA/web/assets/86827803/64402c2e-6d78-49ea-84ff-29939493bc1f)

## To Reproduce
**Affected versions**:
v15

1. Install web_advanced_search from https://github.com/OCA/project/tree/15.0.
3. Activate Debug Mode
4. Go To Apps
5. Under the search field click on "Filters"
6. in the drop down menu click on "Add Advanced Filter"
7. in the wizard change a value in the code editor
8. click anywhere outside of the code editor
9. the values in the upper section stay the same

**Expected behavior**
The values in the code editor should be propagated to the upper section (clickable expression lists).

**Comment by pedrobaeza** [comment]
This is not a problem of the module, but the Odoo core domain widget. @chienandalu do you have the reference for the issue?

**Comment by chienandalu** [comment]
Yes, is a core issue although I couldn't find it in any issue/PR on Odoo

**Comment by coffeinisfun** [comment]
I already have created a PR with a hotfix. It overwrites the relevant method from the domain widget with the v14 code section when using this module
https://github.com/OCA/web/pull/2625

**Comment by sergiocorato** [comment]
> I already have created a PR with a hotfix. It overwrites the relevant method from the domain widget with the v14 code section when using this module #2625

This issue should be closed as PR #2625  has been merged.
